src/components/HTMLEngineProvider/HTMLRenderers/TaskTitleRendered.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import type {CustomRendererProps, TPhrasing, TText} from 'react-native-render-html';
+import {TNodeChildrenRenderer} from 'react-native-render-html';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+function TaskTitleRenderer({tnode}: CustomRendererProps<TText | TPhrasing>) {
```

Comment:
The file name says `TaskTitleRendered`, but class name is `TaskTitleRenderer`. Can we rename the file? as the class name seems more appropriate?